src/main/java/cinema/model/Ticket.java

```diff
@@ -0,0 +1,85 @@
+package cinema.model;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "session_id")
+    private MovieSession movieSession;
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
seems that you should also join fetch user for each ticket when you are getting the shopping cart

Reply:
There is no need to do so since User is already fetched via One-to-One reference in ShoppingCart